pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -539,6 +544,25 @@ public Boolean get() {
         }
     }
 
+    private void generatePropertiesForConfigurationIfNeeded() {
```

Comment:
Since the purpose it's not obvious, can you add a comment on why we need this?

Reply:
Removed